### DISCUSSION TITLE
About Axum permission application

### DISCUSSION THREAD

**Discussion by yuenxillar** [tokio-rs/axum#2873] [Q&A]
### Summary

I want to know how Axum achieves role and permission authentication in web application business, intercepting API requests between the actual function bodies.

### axum version

0.7.5

**Comment by yuenxillar** [comment]
If it exists, please let me know. Thank you

**Comment by mladedav** [comment]
See [ECOSYSTEM.md](https://github.com/tokio-rs/axum/blob/main/ECOSYSTEM.md) if you find there something useful.

**Reply by yuenxillar** [reply]
ok, thank you, Although it's not what I want
